traitlets/traitlets.py

```diff
@@ -2269,95 +2291,110 @@ def validate(self, obj, value):
         self.error(obj, value)
 
 
-class Container(Instance):
-    """An instance of a container (list, set, etc.)
-
-    To be subclassed by overriding klass.
-    """
-    klass = None
-    _cast_types = ()
-    _valid_defaults = SequenceTypes
-    _trait = None
+class Mutable(TypeCast):
```

Comment:
What is Mutable? If it's only here to be a base class, make it private: `_Mutable`. Also make sure that every one of these has a docstring.